modules/ossm-observability-cli.adoc

```diff
@@ -6,7 +6,7 @@
 [id="ossm-observability-cli_{context}"]
 = Accessing metrics and tracing data from the CLI
 
-Access the Jaeger, Prometheus, and Grafana consoles to view and manage your data. 
+Access the Jaeger, Prometheus, and Grafana consoles to view and manage your data.
```

Comment:
Do we need to mention Kiali here?

Reply:
Yes, don't know why it wasn't already mentioned.